keps/sig-autoscaling/4951-configurable-hpa-tolerance/README.md

```diff
@@ -630,18 +633,15 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+N/A.
+
 ###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
 
 <!--
 Pick one more of these and delete the rest.
 -->
 
-- [ ] Metrics
-  - Metric name:
-  - [Optional] Aggregation method:
-  - Components exposing the metric:
-- [ ] Other (treat as last resort)
-  - Details:
+N/A.
```

Comment:
Agree with the N/A here since existing signals from the HPA controller are relevant and can continue to be used.

Maybe we should be explicit though?